backend/web.py

```diff
@@ -95,26 +95,28 @@ def check_vote_table(vote_table):
     num_constituencies = len(vote_table["constituencies"])
 
     if not len(vote_table["votes"]) == num_constituencies:
-        raise ValueError("The data is malformed.")
+        raise ValueError("The vote_table does not match the constituency list.")
     for row in vote_table["votes"]:
         if not len(row) == num_parties:
-            raise ValueError("The data is malformed.")
+            raise ValueError("The vote_table does not match the party list.")
         for p in range(len(row)):
             if not row[p]: row[p] = 0
             if type(row[p]) != int: raise TypeError("Votes must be numbers.")
 
     for const in vote_table["constituencies"]:
         if "name" not in const or not const["name"]:
             raise KeyError(f"Missing data ('vote_table.constituencies[x].name')")
+            name = const["name"]
```

Comment:
oh darn. Wrong indent there. Will fix in next pr

Reply:
As promised:

https://github.com/smari/voting/pull/74/commits/cc00ff0a7c17a8a635daa1a8b90d9c1720f22d49